fly.toml

```diff
@@ -0,0 +1,41 @@
+# fly.toml file generated for keeper on 2022-05-02T06:36:51-05:00
+
+app = "keeper"
+
+kill_signal = "SIGINT"
+kill_timeout = 5
+processes = []
+
+[env]
+  CONF_FILE = "main"
```

Comment:
Just curious, how does the config file actually end up getting its properties? Are the secrets added in the fly console or something?

Reply:
I set them with `flyctl secrets set KEY=value`, but the fly web console works as well I believe